### PR TITLE
Provisioning: Web: add a variable for the admin password

### PR DESCRIPTION
**Pull request by helicopter88**
- Use a script to set the password so that it cannot be read from ansible output
- The password is stored into secrets.yml for production environments
  It can be accessed through ansible-vault

### CODE REVIEW

**Comment by andronat** [comment]
Can you also include a e2e test to check if the admin can login?

**Comment by helicopter88** [comment]
I am looking at the failing test, however it passes locally.

**Comment by helicopter88** [comment]
Tests have been fixed

File: src/klee_web/tests/e2e_tests.js [hunk 1]
```diff
@@ -12,34 +12,51 @@ it('test all', function(done1) {
...
-            .evaluate(getResult, function(actual) {
-                actual.replace(/(?:\r\n|\r|\n)/g, "\n").should.match(expected)
-                testsRun++;
-                if (testsRun == tests.length) {
-                  // notify that all the assertions happened
-                  done1();
-                }
-            }).run();
+
+   for (var i = 0; i < tests.length; i++) {
```
Comment (andronat) [inline src/klee_web/tests/e2e_tests.js]: git report a huge change of lines for this test. If you haven't change anything just changed the indentation I would suggest to revert that and use the same indentation for the second test. It's not obvious to the reviewer what you've changed and bugs could be missed.

**Comment by helicopter88** [comment]
Done

**Comment by ccadar** [comment]
LGTM, thanks

**Pull request merged**